recipe/meta.yaml

```diff
@@ -57,19 +58,35 @@ test:
 
 about:
   home: https://CRAN.R-project.org/package=densityClust
-  license: GPL (>= 2)
-  summary: An improved implementation (based on k-nearest neighbors) of the  density peak clustering
-    algorithm, originally described by Alex Rodriguez  and Alessandro Laio (Science,
+  license: GPL-2.0-or-later
+  summary: An improved implementation (based on k-nearest neighbors) of the density peak clustering
+    algorithm, originally described by Alex Rodriguez and Alessandro Laio (Science,
     2014 vol. 344). It can handle large datasets (> 100, 000 samples) very efficiently.
-    It was initially implemented by  Thomas Lin Pedersen, with inputs from Sean Hughes
-    and later improved by  Xiaojie Qiu to handle large datasets with kNNs.
-  license_family: GPL3
-  license_file: '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
```

Comment:
I'm not sure what to do here. The recipe originally bundled GPL-3. But now the cran skeleton wants to bundle GPL-2. Do we have strong feelings on how to interpret the CRAN license designation "GPL (>=2)"? I've seen conflicting examples, which makes me suspect that the behavior of the CRAN skeleton may have changed over time.

Reply:
answer: https://github.com/conda-forge/r-rtensor-feedstock/pull/8#discussion_r777388072